src/components/DiagramFrame/MetadataPanel/utils.ts

```diff
@@ -116,11 +116,17 @@ export function getFieldMetadata(fields: ILookmlModelExploreField[], selectionIn
   }
 }
 
-export function getViewMetadata(viewResponse: any, lookmlLink: string, selectionInfo: SelectionInfoPacket) {
+export function getViewMetadata(viewResponse: ILookmlModelExplore, isLoading: boolean, lookmlLink: string, selectionInfo: SelectionInfoPacket) {
+  let sqlTableName = "unknown"
+  if (isLoading) {
+    sqlTableName = "Loading..."
```

Comment:
Overloading table name is tacky. I would prefer to seeing returning loading. Means a knock on effect in the component that uses it. From a maintainability standpoint this is bad.

Reply:
Will be addressed in follow up PR